conans/model/editable_cpp_info.py

```diff
@@ -4,54 +4,82 @@
 import os
 import posixpath
 import re
+from collections import defaultdict
 
 import six
 
 
 class EditableCppInfo(object):
+    WILDCARD = "*"
     cpp_info_dirs = ['includedirs', 'libdirs', 'resdirs', 'bindirs']
 
-    def __init__(self, data):
+    def __init__(self, data, uses_namespace):
         self._data = data
+        self._uses_namespace = uses_namespace
 
     @classmethod
-    def create(cls, filepath_or_content, *args, **kwargs):
+    def create(cls, filepath_or_content, require_namespace):
         _parse_func = cls.parse_file if os.path.exists(filepath_or_content) else cls.parse_content
-        data = _parse_func(filepath_or_content, *args, **kwargs)
-        return EditableCppInfo(data)
+        data = _parse_func(filepath_or_content, require_namespace=require_namespace)
+        return EditableCppInfo(data, uses_namespace=require_namespace)
 
     @classmethod
-    def parse_file(cls, filepath, base_path, settings=None, options=None):
-        with open(filepath, 'r') as f:
-            return cls.parse_content(content=six.u(f.read()), base_path=base_path,
-                                     settings=settings, options=options)
+    def parse_file(cls, filepath, require_namespace):
+        with open(filepath) as f:
```

Comment:
Conan uses the ``files.load()`` helper to load text files if possible, not the context + ``six.u`` + bare ``f.read()``.